README.md

```diff
@@ -52,6 +53,17 @@ Pope.prop(data, "nananana")          # nil
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create a new Pull Request
 
+### How to add additional filters Adding more filters is fairly straightforward:
```

Comment:
### How to add additional filters 
Adding more filters is fairly straightforward:

Reply:
ohp, yep, good catch. sorry!